app/io/pathfinder/websockets/ModelTypes.scala

```diff
@@ -6,15 +6,11 @@ import play.api.libs.json._
  * An enum containing the models that a websocket message can use
  */
 object ModelTypes extends Enumeration {
```

Comment:
Why did you get rid of reads and writes? I though that was the whole point of this class.

Please add comments explaining the "new" point of the class.